### PR TITLE
Added Dockerfile and dummy script

### PR DESCRIPTION
**Pull request by Slavko12345**
Added Dockerfile and dummy train script.

### CODE REVIEW

File: ml-service/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM python:3.9.7
+
+COPY train.py /app/
+RUN pip install numpy scikit-learn
```
Comment (truskovskiyk) [inline ml-service/Dockerfile]: please move installation before 

```
COPY train.py /app/
```
reasons:  invalidating the Docker build cache
see full explanation: https://stackoverflow.com/questions/34398632/docker-how-to-run-pip-requirements-txt-only-if-there-was-a-change

Comment (truskovskiyk) [inline ml-service/Dockerfile]: please add versions and put deps to requirements.txt 

reference: https://learnpython.com/blog/python-requirements-file/

**Review by truskovskiyk — COMMENTED** [review]
@Slavko12345 looks good! please address feedback

also - could you add CI yaml file to this PR too?

**Comment by Slavko12345** [comment]
Added requirements, and fixed order of lines in the dockerfile. 
I never thought it is possible to make so many mistakes in literally 3 lines of code, thanks)

**Review by truskovskiyk — APPROVED** [review]
@Slavko12345 amazing!

**Pull request merged**